lib/Jsrt/Jsrt.cpp

```diff
@@ -3910,7 +3910,7 @@ JsErrorCode RunSerializedScriptCore(
         {
             flags = fscrAllowFunctionProxy;
         }
-        if (useParserStateCache)
+        if (useParserStateCache && !CONFIG_FLAG(ForceSerialized))
```

Comment:
so, does this mean that, when forceserialized, we don't want to test parserstatecache?
is that for this particular API, or is that a requirement of the feature (i.e., can't force serialize when we cache parser state)? I might find the answer to this later in the CR